### PR TITLE
[node-bridge] Support streaming response for Serverless Function

### PR DESCRIPTION
**Pull request by TooTallNate**
Adds streaming response support for React Server Components with Next 13.

### CODE REVIEW

**Review by ijjk — APPROVED** [review]

File: packages/node-bridge/bridge.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
+const { URL } = require('url');
```
Comment (styfle) [inline packages/node-bridge/bridge.js:1]: I think you can use the global `URL` instead

File: packages/node-bridge/package.json [hunk 2]
```diff
@@ -23,6 +23,8 @@
   "devDependencies": {
     "@types/aws-lambda": "8.10.19",
     "@types/node": "*",
+    "jsonlines": "0.1.1",
+    "test-listen": "1.1.0",
```
Comment (styfle) [inline packages/node-bridge/package.json:27]: Lets publish a new version of this without all the bloat

**Comment by EndangeredMassa** [comment]
I'm missing a bunch of context to be able to review this in detail. If others have enough, no worries, but if there's somewhere I can read more about the technical details, I'd be happy to dig through it.

File: packages/node-bridge/bridge.js (hunk 1 above)
Comment (TooTallNate) [reply packages/node-bridge/bridge.js:1]: TypeScript doesn't seem to like it, so seems like something isn't configured correctly.

Comment (cb1kenobi) [reply packages/node-bridge/bridge.js:1]: I also ran into not being able to use the global `URL`. Maybe our `@types/node` is out of date?

**Review by styfle — APPROVED** [review]

**Review by ijjk — APPROVED** [review]

**Pull request merged**